src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -3896,29 +3896,6 @@ private void ResetOutputBufferAsNeeded()
             }
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
TabCompletion.Miscellaneous completion tests.Test member completion of a static method invocation
```powershell
Expected a collection with size 34, but got collection with size 33 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 757
757:             $res.CompletionMatches | Should -HaveCount 34
```